gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageFileSystem.java

```diff
@@ -931,19 +931,50 @@ private void repairImplicitDirectory(Future<GoogleCloudStorageItemInfo> infoFutu
    * @param pageToken the page token to list
    */
   public ListPage<FileInfo> listFileInfoForPrefixPage(
-      URI prefix, ListFileOptions listOptions, String pageToken) throws IOException {
+          URI prefix, ListFileOptions listOptions, String pageToken) throws IOException {
```

Comment:
Please, use Google Java Formatter to format changed lines in this PR: https://github.com/google/google-java-format/blob/master/scripts/google-java-format-diff.py